src/shared/parse.cpp

```diff
@@ -4335,7 +4331,7 @@ bool Parse_ReadTokenHandle( int handle, pc_token_t *pc_token )
 	}
 
 	ret = Parse_ReadToken( sourceFiles[ handle ], &token );
-	strcpy( pc_token->string, token.string );
+	memcpy( pc_token->string, token.string, sizeof pc_token->string );
```

Comment:
Should be `Q_strncpyz`? Copying to the maximum buffer length is undefined behavior if it reads unset bytes at the end, as well as less efficient.